libraries/botbuilder-dialogs-adaptive/src/actions/beginSkill.ts

```diff
@@ -85,7 +85,7 @@ export class BeginSkill extends SkillDialog {
 
     /**
      * Called when the dialog is started and pushed onto the dialog stack.
```

Comment:
Should we use the Dialog class reference here?, more lines with this point on this file.